src/pages/Suppliers/index.tsx

```diff
@@ -215,6 +239,60 @@ const SuppliersPage: React.FC = () => {
           </Button>
         </DialogActions>
       </Dialog>
+
+      {/* Edit Supplier Dialog */}
+      <Dialog open={editOpen} onClose={() => setEditOpen(false)}>
```

Comment:
Seperate the dialog into seperate component. Lazy load if possible

Reply:
I did that as well, thanks for addressing this!